app/src/processing/app/Base.java

```
@@ -1335,6 +1340,41 @@ public void selectTargetBoard(TargetBoard targetBoard) {
     onBoardOrPortChange();
     rebuildImportMenu(Editor.importMenu);
     rebuildExamplesMenu(Editor.examplesMenu);
+    try {
+      rebuildRecentBoardsMenu();
+    } catch (Exception e) {
+      // fail silently
```

Why do we need this? Please don't eat exceptions...